features/step_definitions/supplieraz_steps.rb

```diff
@@ -54,33 +68,84 @@
 end
 
 Then(/I am taken to page '(.*)' of results$/) do |page_number|
+  # @TODO: Remove old pagination styles once DM-GOVUK search page is released
   if current_url.include?('suppliers?')
     expect(current_url).to include("#{dm_frontend_domain}/g-cloud/suppliers?")
     expect(current_url).to include("prefix=#{@data_store['supplier_alphabet']}")
     expect(current_url).to include("page=#{page_number}")
     expect(current_url).to include("framework=g-cloud")
     if page_number >= '2'
-      expect(page).to have_selector(:xpath, "//a[contains(text(), 'Previous')]//following-sibling::span[contains(text(),'page')]")
+      expect(page).to have_selector(
+        :xpath,
+        (
+          "//a[contains(text(), 'Previous')]//following-sibling::span[contains(text(),'page')]"
+          "|"
+          "//nav//ul//li//a//span[contains(text(), 'Previous page')]"
```

Comment:
Ruby doesn't concatenate string literals across multiple lines like Python, so this needs a line continuation character to work:
```suggestion
          "//a[contains(text(), 'Previous')]//following-sibling::span[contains(text(),'page')]" \
          "|" \
          "//nav//ul//li//a//span[contains(text(), 'Previous page')]"
```